docs/UserGuide.md

```diff
@@ -91,8 +92,8 @@ A person can have zero or more roles.
 </div>
```

Comment:
Line 91: We should also clarify that while any role can be entered (as long as alphanumeric or contains commas and hyphens), there are certain roles that have a custom format (e.g. athlete, committee member, sponsor, volunteer, etc.) which they can see in the examples.

Reply:
will take note